app/views/spree/admin/variants/_autocomplete.js.erb

```diff
@@ -4,7 +4,7 @@
       {{#if variant.image }}
         <img src='{{variant.image}}' />
       {{ else }}
-        <img src='<%= image_path("noimage/mini.png") %>' />
+        <img src='/noimage/mini.png' />
```

Comment:
but don't we need Rails url helpers so the fingerprinted URL is picked up?

Reply:
I made a slightly different choice for this specific case. These default images are used in various API endpoints, serializers and also in javascript files where generating the fingerprint is a bit messy. Moving them under /public means we can use the non-fingerprinted versions of these images (which is what we were doing before, anyway).

What do you think?

Reply:
I see. That's good enough